config.json

```diff
@@ -1933,6 +1949,14 @@
         "nugetId": "Xamarin.AndroidX.Work.Work.Runtime.Ktx",
         "dependencyOnly": false
       },
+      {
+        "groupId": "com.google.android.libraries.identity.googleid",
```

Comment:
I think we need to bind this in the GPS repository.  The template(s) in this repository will bundle the Java artifact in the NuGet.  However, this library is Android Software Development Kit License, which prohibits us from redistributing the artifact.

Reply:
Dependencies are convoluted if it is there. I cannot reference nuget, because there is not one. I tried but came up with several PRs and that would take too long. 
Let us publish nuget and we can delist it later and continue publishing from GPS-FB-MLKit repo

Reply:
I think when it comes to legal/licensing issues we do not have the flexibility to do it incorrectly now and then fix it later.  ☹️ 

It looks like `androidx.credentials:credentials` does not require this dependency.  We could just bind this one now and worry about the other(s) in the future if they are needed.

Reply:
This is what I get:

```
Dependency errors : 1
1
	System.Exception: 
Artifact
	androidx.credentials.credentials-play-services-auth:1.2.0
has unknown 'Runtime' dependency 
	com.google.android.libraries.identity.googleid.googleid:1.1.0
Either fulfill or exclude this dependency.
```

Reply:
`androidx.credentials:credentials-play-services-auth` contains the dependency, `androidx.credentials:credentials` does not.

If we want to bind them all, we will need to:
- Bind and publish `androidx.credentials:credentials` in this repo
- Bind and publish `com.google.android.libraries.identity.googleid.googleid` in GPS
- Bind and publish `androidx.credentials:credentials-play-services-auth` in this repo

Reply:
Yes. I was there last week and was trying to reduce number of PRs. This was minimal sample (PR) I created with this PR. There are at least 2 more waiting (AX WSU and GPS-FB-MLKit MSU). Both with more fun (updates with new preview dependencies). We will end up freezing everything in config.json with this google's strategy and pace. 

```
<sarcasm-flat-earth-theory>
I have feeling they are doing it on purpose, so we cannot keep up with bindings.
</sarcasm-flat-earth-theory>
```

I need to gather strength for those PRs. Demotivating as hell.